# Exercise 15:  confidence interval for proportions

 

At the lectures we looked an example concerning the opinion poll from February 2017 (cf. slide 2 from the lectures)

We will in this exercise consider this example further.



### Question a)

Of the n=935 persons who were interviewed by Norstat, y = sum(y_i) = 309 would have voted Ap. The following calculations reproduce the result from the lectures (cf. slide 4)

- n = 935
- y = 309
- p = y/n
- se = sqrt(p*(1-p)/n)
- margin = 1.96*se
- lower = p-margin
- upper = p+margin
- cbind(p,margin,lower,upper)


Do the calculations and check that you get the result from the lectures.

In [2]:
n = 935
y = 309
p = y/n
se = sqrt(p*(1-p)/n)
margin = 1.96*se
lower = p-margin
upper = p+margin
cbind(p, margin, lower, upper)

p         margin     lower     upper    
[1,] 0.3304813 0.03015123 0.3003301 0.3606325

From slides, Ap's support are 33.0 % which match the p-value. The margin is 3.0 % which we also got.

### Question b)

In the opinion poll, 122 of the persons interviewed would have voted Fremskrittspartiet (FrP) and 80 would have voted Senterpartiet (Sp). Repeat the calculations above for Fremskrittspartiet and Senterpartiet.

How is the "margin of error" for these parties compared to the "margin of error" for Ap (cf. slide 4)?

In [3]:
n = 935
y = 122
p = y/n
se = sqrt(p*(1-p)/n)
margin = 1.96*se
lower = p-margin
upper = p+margin
cbind(p, margin, lower, upper)

p         margin     lower     upper    
[1,] 0.1304813 0.02159056 0.1088907 0.1520718

The only difference is the parameter y. We get a lower p-value of 13.0 % and a lower margin value of 2.2 % for FrP.

In [4]:
n = 935
y = 80
p = y/n
se = sqrt(p*(1-p)/n)
margin = 1.96*se
lower = p-margin
upper = p+margin
cbind(p, margin, lower, upper)

p         margin     lower      upper    
[1,] 0.0855615 0.01792944 0.06763206 0.1034909

For Sp, we get a p-value of 8.6 % and a margin value of 1.8 % for FrP.